Set minimum content height to prevent cutoff of absolute content

joshuatf opened the pull request:
Fixes #1107 

Sets a minimum height on the content body so that long search results do not get cut off.

### Before
<img width="720" alt="screen shot 2018-12-24 at 2 27 39 pm" src="https://user-images.githubusercontent.com/10561050/50392295-158cb380-0788-11e9-8209-f48653bd96d6.png">

### After
<img width="718" alt="screen shot 2018-12-24 at 2 04 58 pm" src="https://user-images.githubusercontent.com/10561050/50392287-f7bf4e80-0787-11e9-8ad0-274d5ac9ec42.png">

### Detailed test instructions:

1.  Open any report with a search.
2.  Make sure only 1 row is visible to limit content height.
3.  Search using a term that will return at least 10 items.
4.  Note that all items are visible and not cut off.

joshuatf committed:
Set minimum content height to prevent cutoff of absolute content

Aljullu reviewed (approved):
That works as expected. :clap: 

The only small thing I was wondering is whether it would make sense to move the `32px` and `46px` values into variables, maybe inside `client/stylesheets/abstracts/_variables.scss`. That would be useful to self-document these declarations and would make it easier to reuse the values if at some point we need to. What do you think?

joshuatf committed:
Add admin bar height vars for SCSS and apply them

joshuatf commented:
@Aljullu Fantastic idea.  Added those and replaced where I could find references that were used to manipulate distance due to admin bar (e.g., the main header navbar).
